test/IDE/complete_expr_postfix_begin.swift

```diff
@@ -522,11 +522,11 @@ func testInForEach9(arg: Int) {
   let local = 2
   for index in [#^IN_FOR_EACH_9^#:2] {}
   let after = 4
-// NOTE: [Convertible] to AnyHashable.
+// FIXME: '/TypeRelation[Convertible]' to AnyHashable.
 // IN_FOR_EACH_4-NOT: Decl[LocalVar]
-// IN_FOR_EACH_4: Decl[LocalVar]/Local/TypeRelation[Convertible]: local[#Int#];
+// IN_FOR_EACH_4: Decl[LocalVar]/Local: local[#Int#];
 // IN_FOR_EACH_4-NOT: after
-// IN_FOR_EACH_4: Decl[LocalVar]/Local/TypeRelation[Convertible]: arg[#Int#];
+// IN_FOR_EACH_4: Decl[LocalVar]/Local: arg[#Int#];
```

Comment:
This is a regression. For example:
```swift
let _ = [<HERE>: 1]
```
Previously this used to be type checked to `[AnyHashable : Int]`, but now it's `[<<unresolved type>> : Int]`. It's unfortunate, but in practice, I don't think this causes a big difference.